website/docs/sdk-reference/dart.md

```diff
@@ -207,12 +268,54 @@ client.forceRefresh();
 ```
 > `getValue()` returns `defaultValue` if the cache is empty. Call `forceRefresh()` to update the cache.
 
+## Hooks
+
+With the following hooks you can subscribe particular events sent by the SDK:
+
+- `onClientReady()`: This event is sent when the SDK reaches the ready state. When the SDK is configured with lazy load or manual polling it's considered ready right after instantiation.
+When it's using auto polling, the ready state is reached when the SDK has a valid config.json loaded into memory either from cache or from HTTP. When the config couldn't be loaded neither from cache nor from HTTP the `onClientReady` event fires when the auto polling's `maxInitWaitTime` is reached.
```

Comment:
- ~~When it's~~ If the SDK is using polling mode
- ~~When~~ If the config couldn't

Reply:
✅